code/modules/lighting/lighting_atom.dm

```diff
@@ -67,13 +67,13 @@
 
 // Should always be used to change the opacity of an atom.
 // It notifies (potentially) affected light sources so they can update (if needed).
-/atom/proc/set_opacity(new_opacity)
+/*/atom/proc/set_opacity(new_opacity)
 	var/old_opacity = opacity
 	opacity = new_opacity
 	var/turf/T = loc
 	if(old_opacity != new_opacity && istype(T))
 		T.reconsider_lights()
-
+*/
```

Comment:
That breaks lights update.
![default](https://user-images.githubusercontent.com/2458675/47064906-982e4b00-d1fa-11e8-8f5b-80329b935947.PNG)